package.json

```diff
@@ -9,7 +9,9 @@
     "build": "npm run clean && node build.js && tsc --emitDeclarationOnly --outDir lib",
     "clean": "rimraf dist",
     "lint": "eslint .",
-    "test": "jest --coverage"
+    "test": "jest --coverage",
+    "installAllDependencies": "npm install && npm run build && npm install --prefix docs",
+    "docs": "npm run docs --prefix docs"
```

Comment:
Can we just combine these two? I tried it locally and it's a nice DX. I don't think there's much time saved by splitting out the two commands.

```
"docs": "npm install && npm run build && npm install --prefix docs && npm run docs --prefix docs"
```

Reply:
https://github.com/vectara/react-chatbot/pull/16/commits/2d6ae3df1d36d0eed7df2c3ded21be8a486b8dcf